### ISSUE TITLE
[Image Modification] WebP conversion stopped working

### ISSUE THREAD

**Issue by vivgui** [raycast/extensions#10438]
### Extension

https://www.raycast.com/HelloImSteven/sips

### Description

Today I tried using the extension to convert a few images to WEBP and got the following error:

```
Command failed: chmod +x /Users/vivgui/.config/raycast/extensions/7c158f0e-b70b-4922-8a1b-5c8a9ccb55d6/assets/webp/cwebp 
chmod: /Users/vivgui/.config/raycast/extensions/7c158f0e-b70b-4922-8a1b-5c8a9ccb55d6/assets/webp/cwebp: No such file or directory
```

Maybe the recent [update](https://github.com/raycast/extensions/pull/10418) broke it?

### Steps To Reproduce

1. In MacOS, select an image in finder
2. Open Raycast and select the Image Modification extension
3. Use the "Convert Images" command
4. Select the "WEBP" format
5. See error 
```
Command failed: chmod +x /Users/vivgui/.config/raycast/extensions/7c158f0e-b70b-4922-8a1b-5c8a9ccb55d6/assets/webp/cwebp 
chmod: /Users/vivgui/.config/raycast/extensions/7c158f0e-b70b-4922-8a1b-5c8a9ccb55d6/assets/webp/cwebp: No such file or directory
```

**Comment by SKaplanOfficial** [comment]
Will be fixed once #10435 gets merged!

**Comment by vivgui** [comment]
ok sounds good!
